app/presenters/outcome_presenter.rb

```diff
@@ -10,45 +10,28 @@ def initialize(i18n_prefix, node, state = nil, options = {})
   end
 
   def title
-    if use_template? && title_erb_template_exists?
+    if title_erb_template_exists?
       render_erb_template
       title = @view.content_for(:title) || ''
       strip_leading_spaces(title.chomp)
-    else
-      translate!('title')
     end
   end
 
-  def translate!(subkey)
-    output = super(subkey)
-    if output
-      output.gsub!(/\+\[data_partial:(\w+):(\w+)\]/) do |match|
-        render_data_partial($1, $2)
-      end
-    end
-
-    output
-  end
-
   def body(html: true)
-    if use_template? && body_erb_template_exists?
+    if body_erb_template_exists?
       render_erb_template
       govspeak = @view.content_for(:body) || ''
       govspeak = strip_leading_spaces(govspeak.to_str)
       html ? GovspeakPresenter.new(govspeak).html : govspeak
-    else
-      super
```

Comment:
I don't think we can raise an exception as none of title, body or next steps are required to be present in the Outcomes. The [calculate-agricultural-holiday-entitlement done outcome][done-outcome] is an example of one that only has a title and no body.

[done-outcome]: https://github.com/alphagov/smart-answers/blob/master/lib/smart_answer_flows/calculate-agricultural-holiday-entitlement/done.govspeak.erb